tests/e2e/reconcile/chainsaw-test.yaml

```diff
@@ -0,0 +1,60 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: reconcile
+spec:
+  steps:
+  - name: step-00
+    try:
+    - apply:
+        file: 00-install-storage.yaml
+    - assert:
+        file: 00-assert.yaml
+  - name: step-01
+    try:
+    - apply:
+        file: 01-install-tempo.yaml
+    - assert:
+        file: 01-assert.yaml
+  - name: step-02
+    try:
+    - command:
+        args:
+        - delete
+        - service
+        - -n
+        - $NAMESPACE
+        - tempo-simplest-querier
+        entrypoint: kubectl
+    - assert:
+        file: 02-assert.yaml
+  - name: step-03
+    try:
+    - apply:
+        file: 03-update-storage-secret.yaml
+    - assert:
+        file: 03-assert.yaml
+  - name: step-04
+    try:
+    - command:
+        args:
+        - -c
+        - 'kubectl get --namespace $NAMESPACE configmap tempo-simplest -o jsonpath="{.data.tempo\\.yaml}"
```

Comment:
Is there a way that this could be replaced with an inline assertion tree? (Can also be a follow-up PR, just curious if we can replace this kuttl workaround with some of the new features in chainsaw)

Reply:
Definitely

Reply:
(will do it in follow-up BTW)